local.js

```diff
@@ -66,14 +66,16 @@ function adjustClockSkew(res) {
 	CLOCK_SKEW = serverTime - currentTime;
 }
 
-function requestToken(scope) {
+function requestToken(scope, doNotKeepSessionAlive) {
+	const payload = doNotKeepSessionAlive
+		? {scope: scope, 'X-D2L-Session': 'no-keep-alive'}
+		: {scope: scope};
```

Comment:
Building on this suggestion: https://github.com/Brightspace/frau-jwt/pull/39/files#r849800224

```suggestion
function requestToken(scope, opts) {
	const params = { scope: scope };
	
	if (opts && opts.extendSession === false) {
		params['X-D2L-Session'] = 'no-keep-alive';
	}
```